<!doctype html>
<html>
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
  <title>Current Topics in Epigenomics</title>
  <link rel="stylesheet" href="http://cdn.amazeui.org/amazeui/2.6.1/css/amazeui.min.css">
  <script src="http://libs.useso.com/js/jquery/2.1.1/jquery.min.js"></script>
  <script src="http://cdn.amazeui.org/amazeui/2.6.1/js/amazeui.min.js"></script>
  <style>
    ol.lower-alpha { list-style-type:lower-alpha; }
  </style>
</head>
<body>
  <header class="am-margin am-text-center">
    <div class="am-text-xl">
      系统生物学与生物信息学<br/>
      海外学者短期讲学系列课程
    </div>
    <h1 class="am-text-xxl am-margin-top">
      Current Topics in Epigenomics<br/>
      表观基因组前沿
    </h1>
    <hr/>
  </header>

  <div class="am-container">
    <h2 class="am-margin-top-xl">Instructor</h2>
    <p><a href="http://wang.wustl.edu/people" target="_blank">Ting Wang（王艇）</a>, Washington University in St. Louis, School of Medicine, Associate Professor</p>

    <h2>Contact</h2>
    <ul>
      <li>Dr.Jin Gu (FIT 1-112)</li>
      <li>Aidi Tan (Teaching Assistant, FIT 1-108, aidistan@163.com)</li>
    </ul>

    <h2>Time & Place</h2>
    <p>9:50~12:15, 4/15, 4/18, 4/20, 4/22, 4/25 4/27, in Main Building（中央主楼）511, Tsinghua University</p>

    <h2>Description</h2>
    <p>Epigenomics is an emerging frontier of science. This short course will cover current topics in epigenomics, including modern epigenomics technologies and their data analysis in the context of integration of big bio-data.</p>

    <h2>Syllabus</h2>
    <ol>
      <li>Genes, genome, and modern genomics (Apr 15)
        <ol class="lower-alpha am-margin-0">
          <li>Gene regulation</li>
          <li>Current bioinformatics and genomics</li>
          <li>High-throughput sequencing</li>
        </ol>
      </li>
      <li>Epigenetic mechanisms (Apr 18)
        <ol class="lower-alpha am-margin-0">
          <li>DNA methylation</li>
          <li>Histone code</li>
          <li>RNA components of the epigenome</li>
          <li>Interplay between genome and epigenome</li>
        </ol>
      </li>
      <li>Epigenome and life (Apr 20)
        <ol class="lower-alpha am-margin-0">
          <li>Development</li>
          <li>Health and diseases</li>
          <li>Epigenetics in cancer</li>
          <li>Environmental epigenomics</li>
        </ol>
      </li>
      <li>Technology in modern epigenomics (Apr 22)
        <ol class="lower-alpha am-margin-0">
          <li>Epigenomic approaches</li>
          <li>Sequencing based technology</li>
          <li>Analysis of epigenomic data</li>
        </ol>
      </li>
      <li>From genome to epigenome to genome 4D (Apr 25)
        <ol class="lower-alpha am-margin-0">
          <li>Chromatin biology</li>
          <li>Genome and epigenome architecture</li>
          <li>Epigenetic annotation of the genome</li>
          <li>Higher order of the genome</li>
        </ol>
      </li>
      <li>Modern epigenomic resources – epigenome and big bio-data (Apr 27)
        <ol class="lower-alpha am-margin-0">
          <li>ENCODE</li>
          <li>Roadmap epigenomics</li>
          <li>4D nucleome network</li>
          <li>The international epigenomics community</li>
        </ol>
      </li>
    </ol>

    <h2>Lab assignment (optional)</h2>
    <p>Instructor will design a small project involving using real life, high-throughput sequencing data to analyze patterns of the epigenome in the context of normal human cell type differentiation. The goal of the project is to walk through a typical data analysis pipeline in epigenomics study, from mapping reads, to calling peaks, to compute correlations and other statistics of the data, and to visualize the results. Everyone will start with raw reads, and can be provided any intermediate results as input for the next step. We can define the whole problem on the first day and allow students to carry it as a project to different levels.</p>

    <h2>Workshop (optional)</h2>
    <p>Instructor will present a workshop to introduce epigenomic resources including the UCSC Genome Browser and the WashU Epigenome Browser. Students will learn how to access, download, analyze, and visualize epigenomic data using these tools.</p>

    <h2>Journal club (optional)</h2>
  	<p>Students will be assembled into small groups, and assigned a classic paper in the field. Students will then present a journal club and lead discussions. Instructor will provide feedback.</p>
  </div>

  <footer class='am-footer am-footer-default am-margin-top'>
    <hr/>
    <div class='am-footer-miscs'>生物信息学教育部重点实验室</div>
    <div class='am-footer-miscs'>清华信息国家实验室（筹）生物信息学研究部/系统与合成生物学中心</div>
    <div class='am-footer-miscs'>清华大学自动化系计算生物学实验室</div>
  </footer>
</body>
</html>
